build-nginx.sh

```diff
@@ -21,7 +21,7 @@ source_nginx=https://nginx.org/download/
 # Look up latest versions of each package
 version_pcre=$(curl -sL ${source_pcre} | grep -Eo 'pcre\-[0-9.]+[0-9]' | sort -V | tail -n 1)
 version_zlib=$(curl -sL ${source_zlib} | grep -Eo 'zlib\-[0-9.]+[0-9]' | sort -V | tail -n 1)
-version_openssl=$(curl -sL ${source_openssl} | grep -Eo 'openssl\-[0-9.]+[a-z]?' | sort -V | tail -n 1)
+version_openssl=$(curl -sL ${source_openssl} | grep -Eo 'openssl\-[0-9]+\.[0-9]+\.[0-9]+[a-z]?(?=\.tar\.gz)' | sort -V | tail -n 1)
```

Comment:
I think in order to enable the positive lookahead feature, we need to switch from 'extended' to 'Perl' style regex in grep (`grep -Po` instead of `grep -Eo`), so the whole line is now:
```
version_openssl=$(curl -sL ${source_openssl} | grep -Po 'openssl\-[0-9]+\.[0-9]+\.[0-9]+[a-z]?(?=\.tar\.gz)' | sort -V | tail -n 1)
```

Reply:
I've made the change, thank you.